### ISSUE TITLE
Show route in vehicle tooltips and popups

### ISSUE THREAD

**Issue by roata** [roataway/web-ui#102]
Since we added the ability to observe multiple routes at a time, it is now necessary to show the route number when a vehicle is examined. Otherwise - it is not clear what route the vehicle belongs to.

![image](https://user-images.githubusercontent.com/48152696/71493543-d59b5900-283f-11ea-8db1-19199bd1b206.png)

The pop-up should say:

```
Route: 22
Board: 1268
```

The same message should be shown on the tooltip of a vehicle (on mouse-over).

**Comment by ralienpp** [comment]
@dasshield, can you add your change to these pop-ups too?
